ch_backup/util.py

```diff
@@ -41,6 +41,7 @@
     timedelta(seconds=-1 * (time.altzone if time.daylight else time.timezone))
 )
 _ALLOWED_NAME_CHARS = set(["_"] + list(ascii_letters) + list(digits))
+_UNUSUAL_CHARS_REGEX = re.compile(r"[^\u0020-\u007e]")
```

Comment:
After the [comment](https://github.com/yandex/ch-backup/pull/151/files#r1594025940), im think we don't need it. But if you will keep it:

Actually it isn't clear for me what does this mean.  Do we really need to use unicode here? Why not ASCII? As I can see it is just ASCII Punctuation & Symbols chars. See (https://en.wikipedia.org/wiki/List_of_Unicode_characters)

I think it is better to use simple for loop with  `string.punctuation`. Code will be more readable. Can we do it like that?